projects/rawkode.academy/website/astro.config.ts

```diff
@@ -4,42 +4,57 @@ import react from "@astrojs/react";
 import sitemap from "@astrojs/sitemap";
 import tailwind from "@astrojs/tailwind";
 import vue from "@astrojs/vue";
+import expressiveCode from "astro-expressive-code";
 import { defineConfig, envField } from "astro/config";
 import rehypeExternalLinks from "rehype-external-links";
+import rehypeMermaid from "rehype-mermaid";
 
 export default defineConfig({
-  output: "server",
-  integrations: [mdx(), react(), sitemap(), tailwind(), vue()],
-  adapter: cloudflare({
-    imageService: "cloudflare",
-  }),
-  site: import.meta.env.DENO
-    ? "http://localhost:4321"
-    : "https://rawkode.academy",
-  env: {
-    validateSecrets: true,
-    schema: {
-      ZITADEL_URL: envField.string({
-        context: "server",
-        access: "public",
-      }),
-      ZITADEL_CLIENT_ID: envField.string({
-        context: "server",
-        access: "public",
-      }),
-    },
-  },
-  security: {
-    checkOrigin: true,
-  },
-  markdown: {
-    rehypePlugins: [
-      [
-        rehypeExternalLinks,
-        {
-          target: "_blank",
-        },
-      ],
-    ],
-  },
+	output: "server",
+	integrations: [
+		expressiveCode({
+			themes: ["catppuccin-mocha", "catppuccin-latte"],
+		}),
+		mdx(),
+		react(),
+		sitemap(),
+		tailwind(),
+		vue(),
+	],
+	adapter: cloudflare({
+		imageService: "cloudflare",
+	}),
+	site: import.meta.env.DENO
```

Comment:
a side note: This might not work for everyone. `import.meta.env.DEV` might be better suitable